### PR TITLE
fix(image): fix cropper overflow

### PR DESCRIPTION
**Pull request by PatrickMi**

### CODE REVIEW

**Comment by dargmuesli** [comment]
Which issue does this PR intend to resolve? You can add it with "Resolves #xxx" or via the side menu option "development" :+1:

File: nuxt/components/_/ImageUploadGallery.vue [hunk 1]
```diff
@@ -100,13 +100,15 @@
...
       :submit-task-provider="() => getUploadBlobPromise()"
     >
-      <Cropper
-        ref="cropperRef"
-        :src="fileSelectedUrl"
-        :stencil-props="{
-          aspectRatio: 1 / 1,
-        }"
-      />
+      <div class="h-80 border-">
```
Comment (dargmuesli) [inline nuxt/components/_/ImageUploadGallery.vue:103]: `border-` should not exist, does it?

**Comment by dargmuesli** [comment]
I've made use of the pretty undocumented function [`initStretcher`](https://advanced-cropper.github.io/vue-advanced-cropper/algorithms/init-stretcher.html) to replace the dimension calculation.
I also reverted unrelated changes and opened a separate PR for those: #1009.

**Review by dargmuesli — APPROVED** [review]

**Review by PatrickMi — COMMENTED** [review]
Looks good to me!

**Pull request merged**